### DISCUSSION TITLE
ImageMagick on Windows 11

### DISCUSSION THREAD

**Discussion by AlanSWest** [ImageMagick/ImageMagick#5547] [Help]
I have loaded ImageMagick-7.1.0-Q16-HDRI onto my new Windows 11 laptop. This I use in conjunction with LR/Mogrify.

Whenever I export a file from lightroom a internet explorer page opens on this page - file:///C:/Program%20Files/ImageMagick-7.1.0-Q16-HDRI/www/api/mogrify.html.

When I close the internet explorer window the export continues without a problem.

Any ideas how I can overcome this?

Kind Regards
Alan West

**Comment by dlemstra** [comment]
This is probably a Lightroom issue?
